substrate/bin/node/runtime/src/lib.rs

```diff
@@ -1528,6 +1528,27 @@ where
 	type RuntimeCall = RuntimeCall;
 }
 
+impl<C> frame_system::offchain::CreateAuthorizedTransaction<C> for Runtime
```

Comment:
We should add the new extension at least to `kitchensink-runtime`, if not the other test runtimes (but probably there too). Without it, this trait impl wouldn't make sense anyway.

Reply:
Yes I forgot, I added it everywhere in https://github.com/paritytech/polkadot-sdk/pull/6324/commits/25691b8e945a6453fa65fd5ae6eb54b6a523f280